home/home_2.1/HelloWorld.cpp

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <unordered_map>
+#include <sstream>
+
+const std::string FILENAME = "user_stats.txt";
+
+std::unordered_map<std::string, int> readStats() {
+    std::unordered_map<std::string, int> stats;
+    std::ifstream infile(FILENAME);
+    std::string line;
+    while (std::getline(infile, line)) {
+        std::istringstream iss(line);
+        std::string name;
+        int count;
+        if (iss >> name >> count) {
+            stats[name] = count;
+        }
+    }
+    return stats;
+}
+
+void writeStats(const std::unordered_map<std::string, int>& stats) {
+    std::ofstream outfile(FILENAME, std::ios::trunc);
+    for (const auto& pair : stats) {
+        outfile << pair.first << " " << pair.second << "\n";
+    }
+}
+
+void deleteStats() {
+    std::ofstream outfile(FILENAME, std::ios::trunc);
+}
+
+void displayUsage() {
+    std::cerr << "Usage: <name> or <name> delete or bread\n";
+}
+
+int main() {
+    std::unordered_map<std::string, int> stats = readStats();
+    
+    while (true) {
+        std::cout << "Enter input: ";
+        std::string input;
+        std::getline(std::cin, input);
+
+        if (input.empty()) {
+            std::cerr << "Input cannot be empty. Please try again.\n";
+            continue;
+        }
+
+        std::istringstream iss(input);
+        std::string name, command;
+        iss >> name >> command;
+
+        if (name == "bread") {
```

Comment:
use constexpr constants with names instead of hardcoding across the code. for example you have 2 places with literal delete but what if someone will decide to change delete to clear word - then you will need to search across the code for all the usages instead of single line replacement.